moped-editor/src/views/projects/projectsListView/useProjectListViewQuery/useAdvancedSearch.js

```diff
@@ -84,6 +85,15 @@ const makeAdvancedSearchWhereFilters = (filters) =>
           return null;
         }
       }
+      // If we are filtering on a date equality we need to create a custom search string to handle the date/timestampz conversion
+      if (type === "date" && gqlOperator === "_eq") {
+        const nextDay = format(
+          addDays(parseISO(value.replaceAll('"', "")), 1),
+          "yyyy-MM-dd"
+        );
+        // Greater/equal to the selected day and less than the next day will return all timestampz for the given date
+        return `_and: [ { ${field}: { ${`_gte`}: ${value} } }, { ${field}: { ${`_lt`}: "${nextDay}" } } ]`;
```

Comment:
in order to be able to search for all timestampz on a given date we need to create this greater than/less than query to use instead

Reply:
nice fix! this behavior was really surprising to me, since pg SQL will generously handle the casting for you when you do a query like `where modified_date = '2024-04-01'`.  cool that Chia was able to find an open Hasura issue for it 👍

Reply:
+1 - this is great! 👏 I also suspect that we will find ourselves in this part for https://github.com/cityofaustin/atd-data-tech/issues/9418. Really nice to be reminded where this happens. 🙏